modoboa/admin/constants.py

```diff
@@ -99,3 +99,9 @@
     ("domain", _("Domain")),
     ("relaydomain", _("Relay domain")),
 ]
+
+DKIM_KEY_LENGTHS = [
+    (1024, "1024"),
```

Comment:
1024 key must be insecure https://gsuiteupdates.googleblog.com/2016/05/getting-rid-of-spoofers-digitally-sign.html

Reply:
I disagree, 1024 bit is **less** secure not **in**secure.  1024 bit should still be available (not the default) as there are some DNS providers that still don't support >1024 bit keys, yes it's 2018 but progress is slow 😞

Reply:
Good point. So I'll leave it for now.